nautobot/dcim/api/serializers.py

```diff
@@ -1083,6 +1083,44 @@ def validate(self, data):
         return super().validate(data)
 
 
+class InterfaceSerializerVersion13(InterfaceSerializer):
+    parent = NestedInterfaceSerializer(required=False, allow_null=True)
+    bridge = NestedInterfaceSerializer(required=False, allow_null=True)
```

Comment:
Given these are new, optional fields, I would expect that they should be backwards-compatible and not require introducing a versioned serializer. Am I missing something here?